server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -88,71 +115,17 @@ func (r *Runner) Run(ctx workflow.Context) error {
 	if planReview.Status == Rejected {
 		return nil
 	}
+
 	// Run apply steps
-	for _, step := range r.Request.Root.Apply.Steps {
-		err := r.runStep(ctx, step)
-		if err != nil {
-			return errors.Wrap(err, "running step")
-		}
+	_, err = r.JobRunner.Run(ctx, r.Request.Root.Apply, rootInstance)
+	if err != nil {
+		return errors.Wrap(err, "running step")
```

Comment:
```suggestion
		return errors.Wrap(err, "running apply job")
```

same for the other statement